.golangci.yml

```diff
@@ -24,6 +24,7 @@ issues:
     - "G108: Profiling endpoint is automatically exposed on /debug/pprof"
 run:
   timeout: 10m
+  tests: false
```

Comment:
Why did we do this?

Reply:
Mainly to stop linting failures in tests....shall i revert it?

Reply:
Its a bit lazy but it sops having to add `//nolint` in tests

Reply:
If we're doing it in lots of places, I guess that's fine.